P5/Source/Specs/teidata.version.xml

```diff
@@ -9,21 +9,20 @@ See the file COPYING.txt for details
           ident="teidata.version">
   <desc versionDate="2013-11-20"
          xml:lang="en">defines the range of attribute values which may be used to
-  specify a TEI or Unicode version number.</desc>
+    specify a version number following the Semantic Versioning Specification.</desc>
   <desc xml:lang="fr"
          versionDate="2007-06-12">définit la gamme des valeurs d'attribut
   exprimant un numéro de version TEI.</desc>
   <content>
-      <dataRef name="token" restriction="[\d]+(\.[\d]+){0,2}"/>
+    <dataRef name="token" restriction="(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-((0|[1-9]\d*|\d*[\-a-zA-Z][\-0-9a-zA-Z]*)(\.(0|[1-9]\d*|\d*[\-a-zA-Z][\-0-9a-zA-Z]*))*))?(\+([\-0-9a-zA-Z]+(\.[\-0-9a-zA-Z]+)*))?"/>
    </content>
-  <remarks versionDate="2012-06-17"
+  <remarks versionDate="2020-05-04"
             xml:lang="en">
       <p>The value of this attribute follows the pattern specified by
-    the Unicode consortium for its version number (<ptr target="http://unicode.org/versions/"/>). A version number
-    contains digits and fullstop characters only. The first number
-    supplied identifies the major version number. A second and third
-    number, for minor and sub-minor version numbers, may also be
-    supplied.
+        the Semantic Versioning Specification (<ptr target="https://semver.org"/>). A version number
+    contains digits only and consists at least of three parts separated by a dot. 
```

Comment:
1. “consists at least of three parts” should probably be “consists of at least three parts”.
2. But in truth (as alluded to above), I am not sure why a user should be **required** to specify the patch number in the `@version` of `<TEI>` or `<teiCorpus>`.